andela/hc-silicon
# #157694766 Create scheduled tasks

#### What does this PR do?
- Add a section on the dashboard for Tasks where a user can send an email and backup data on a CSV file

#### Description of Task to be completed?
When a user goes to Tasks section he/she can send an email from the healthschecks. A user can also backup the summary of his checks or logs of a specific function on a CSV file that is downloaded automatically. 
#### How should this be manually tested?
- Start the server by running the command `./manage.py runserver` and access the `URL http://localhost:3000.`
- Go to Tasks tab:-
  - To send an email enter the recipient email, subject and body then click send button
  - To backup checks summary select, `All checks` then enter the name of the file to backup the data.
  - To backup check ping logs select, check's name then enter the name of the file to backup the data.
#### What are the relevant pivotal tracker stories?
[Create scheduled tasks](https://www.pivotaltracker.com/story/show/157694766)

Screenshots
A sample of the dashboard where the user will access the tasks
![image](https://user-images.githubusercontent.com/22454909/41730272-19ae45aa-7584-11e8-9a4a-3e51b18942ce.png)

A sample of an email sent
![image](https://user-images.githubusercontent.com/22454909/41729395-be91290a-7581-11e8-86a4-c552524ad5f1.png)

A sample of checks summary backed up on CSV file
![image](https://user-images.githubusercontent.com/22454909/41729594-462f51fc-7582-11e8-99cb-654a54f08064.png)

A sample of ping logs of a check backed up on CSV file
![image](https://user-images.githubusercontent.com/22454909/41729699-90251c9c-7582-11e8-90f5-fd8c52fec801.png)


@Awesome94  @Felistas  @daktari01 @kzyGit  @tonykimathi  @muhozi
